src/Models/Address.php

```diff
@@ -0,0 +1,161 @@
+<?php namespace DivideBV\PHPDivideIQ\Models;
+
+/**
+ * Created by PhpStorm.
+ * User: Nav
+ * Date: 28-07-16
+ * Time: 12:14
+ */
```

Comment:
Removed and replaced in this commit: https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/19da445b970041225a00f424ad3f2597142fb493